setup.py

```diff
@@ -24,7 +24,7 @@
           'chardet>=1.0.1',
           'cryptography>=0.5',
           'idna>=2.5',
-          'ply>=3.10',
+          'ply==3.10',
```

Comment:
Why is this version pinned?

Reply:
Because if the version of PLY were to change then the format of the generated parsetab files could change requiring them to be regenerated.

Reply:
Pinning version in setup.py is not a good idea. Instead I suggest adding logic that checks if the installed ply version is the one that the tab files were generated with. And if it is not, then regenerate them automatically.

Reply:
Actually I just glanced through the source of PLY and it looks like it uses pickle and checks the version of PLY it was generated with, so I think we don't need to pin it.

Reply:
See https://github.com/dabeaz/ply/blob/master/ply/yacc.py#L2012 for the code I'm referring to.

Reply:
@mhahnenberg, you are correct that PLY will regenerate the parsetab files if it notices that they're out of date, but the way in which it generates those parsetab files is not safe to do if you have multiple python processes running the same code at the same time. See https://github.com/mailgun/flanker/issues/206 for more information.